.coafile

```diff
@@ -28,6 +28,11 @@ bears = PyUnusedCodeBear
 default_actions =
     PyUnusedCodeBear: ApplyPatchAction
 
+[commit]
+bears = GitCommitBear
+shortlog_trailing_period = false
+shortlog_regex = ([^:]*: [A-Z].*)
```

Comment:
Nope, just my interpretation of the guidelines. Plus, [^:]* seems redundant with [^:]+ . Should I make mine conform?